templates/form-fields/file-field.php

```diff
@@ -32,19 +32,20 @@
 <div class="wpum-uploaded-files">
 	<?php if ( ! empty( $data->value ) ) : ?>
 		<?php if ( is_array( $data->value ) ) : ?>
-			<?php foreach ( $data->value as $value ) : ?>
+			<?php 	if (isset($data->value['url']) || array_key_exists('url', $data->value)) {
```

Comment:
Please keep the formating to the existing style (based on https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/) - maybe your code editor / IDE can autoformat for you

Reply:
The code formatting is more than just line alignment, its spaces between functions and parenthesis etc.